scripts/setup-centos8.sh

```diff
@@ -132,6 +140,38 @@ function install_protobuf {
   )
 }
 
+function install_grpc {
+  wget_and_untar https://github.com/grpc/grpc/archive/refs/tags/v1.48.1.tar.gz grpc
+  (
+    cd grpc
+    cmake_install -DgRPC_BUILD_TESTS=OFF -DgRPC_ABSL_PROVIDER=package -DgRPC_ZLIB_PROVIDER=package -DgRPC_CARES_PROVIDER=package -DgRPC_RE2_PROVIDER=package -DgRPC_SSL_PROVIDER=package -DgRPC_PROTOBUF_PROVIDER=package -DgRPC_INSTALL=ON
+  )
+}
+
+function install_crc32c {
+  wget_and_untar https://github.com/google/crc32c/archive/refs/tags/1.1.2.tar.gz crc32c
+  (
+    cd crc32c
+    cmake_install -DBUILD_SHARED_LIBS=OFF -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCRC32C_BUILD_TESTS=OFF -DCRC32C_BUILD_BENCHMARKS=OFF -DCRC32C_USE_GLOG=OFF
+  )
+}
+
+function install_nlohmann_json {
+  wget_and_untar https://github.com/nlohmann/json/archive/refs/tags/v3.11.3.tar.gz nlohmann_json
+  (
+    cd nlohmann_json
+    cmake_install -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DBUILD_TESTING=OFF -DJSON_BuildTests=OFF
+  )
+}
+
+function install_google_cloud_cpp {
+  wget_and_untar https://github.com/googleapis/google-cloud-cpp/archive/refs/tags/v2.24.0.tar.gz google_cloud_cpp
+  (
+    cd google_cloud_cpp
+    cmake_install -DBUILD_TESTING=OFF -DGOOGLE_CLOUD_CPP_ENABLE_EXAMPLES=OFF -DGOOGLE_CLOUD_CPP_ENABLE=storage,bigquery
+  )
+}
+
```

Comment:
Why is this here? it (and it's dependencies) should be in setup-adapters.sh

Reply:
At the moment, it is the only option to have the CI build those libraries.

Reply:
My guess is that there is some magic trick to re-trigger a new docker image, but I do not know that trick yet.